### DISCUSSION TITLE
Did the Twitch follow API change break notifications?

### DISCUSSION THREAD

**Discussion by sfkpmr** [streamlink/streamlink-twitch-gui#821] [Q&A] [answered]
I've not seen any notifications for a bit, which I used to do. Is this related to the API change mentioned in the latest update?

I did update, but still no notifications.

**Comment by bastimeyer** [comment, marked as answer]
No. Twitch has removed the `CREATE` and `DELETE` verbs on the follow channel API endpoints, which is completely unrelated to the notifications service, which polls the list of streams of all followed channels on an entirely different API endpoint.

If you're not seeing any notifications, then you either are not logged in, have notifications disabled, have notifications paused, selected an invalid notification provider, disabled notifications in your OS settings, your OS is incompatible or Twitch has responded with three errors in a row and Streamlink Twitch GUI stopped polling. These are the possibilities when not seeing any notifications despite followed channels going live. There's a button in the notifications settings menu which lets you test the notifications. If this isn't working for you, then you'll have to figure out your OS settings. Streamlink Twitch GUI will log notification attempts to the debug channel when started with the `--loglevel=debug` command line parameter., so you can check that as well. Take a look at the wiki.

Without any details about your system and configuration though I won't be able to help.

**Comment by sfkpmr** [comment]
Thanks, will do a proper bug report if it continues. For now at least some notifications are coming through.
